addons/report/models/report.py

```diff
@@ -194,6 +194,8 @@ def get_pdf(self, docids, report_name, html=None, data=None):
         # Minimal page renderer
         view_obj = self.env['ir.ui.view']
         render_minimal = partial(view_obj.with_context(context).render_template, 'report.minimal_layout')
+        # Page header renderer - to show the link in alert box if address and account number is not configured. This should be added in header but because of some issue on git hub, adding here.
+        render_minimal_page_header = partial(view_obj.with_context(context).render_template, 'report.external_header_page')
```

Comment:
According to spec, there is a requirement to show the hyperlinks in PDF header and footer. But because of one git hub issue (mentioned in task) we are unable to show the hyperlink. That is why showing one alert box after header and before report body.
To show alert messages new template has been created and have rendered from here so that it can display before body.